package.json

```diff
@@ -42,11 +42,13 @@
     "@feathersjs/express": "^1.3.1",
     "@feathersjs/feathers": "^3.3.1",
     "@feathersjs/socketio": "^3.2.9",
+    "async": "^2.6.2",
```

Comment:
is this used?

Reply:
this is an manually required package of another package